package.json

```diff
@@ -4,12 +4,13 @@
   "description": "This package provides Neolution's .eslintrc as an extensible shared config.",
   "homepage": "https://github.com/neolution-ch/eslint-config-neolution",
   "main": "index.js",
+  "type": "module",
   "exports": {
-    ".": "./index.js"
+    ".": "./src/index.mjs",
+    "./package.json": "./package.json"
   },
   "scripts": {
-    "prelint": "git ls-files | xargs eclint check",

```

Comment:
here you dropped the eclint check, if we don't want it anymore we can also uninstall the package

Reply:
@neotrow not sure, is exlint still needed? or should this be covered with the prettier now?

Reply:
eclint validates that the files follows the rules in .editorconfig, but that tool is really old and not maintained anymore, I suggest to remove it completely

Reply:
maybe something like this?
https://www.npmjs.com/package/eslint-plugin-editorconfig
or was this already considered?